configs/platforms/windows-2012r2-x64.rb

```diff
@@ -21,5 +21,5 @@
 
   plat.platform_triple "x86_64-unknown-mingw32"
 
-  plat.package_type "nuget"
+  plat.package_type "msi"
```

Comment:
Also, it should just default to building msis, so you should be able to just remove this line. It's not a bad thing to keep in in though, for extra clarity